src/plugin.js

```diff
@@ -740,6 +743,35 @@ const contribAdsPlugin = function(options) {
 
   };
 
+  /**
+   * iOS Safari will change caption mode to 'showing' if a user previously
+   * turned captions on manually for that video source, so this TextTrackList
+   * 'change' event handler will re-disable them in case that occurs during ad playback
+   */
+  const iOSTrackListChangeHandler = function() {
```

Comment:
At this point the name is a misnomer, we could call it something like `textTrackChangeHandler`

Reply:
I believe it could mislead a reader into thinking it is only called on iOS, which is not the case.

Reply:
We could rename it to the more general `textTrackChangeInAdsHandler`, what do you think?

Reply:
The handler is always present even if it's not in ad mode, and the check for ads is within the handler itself. `textTrackChangeHandler` is still the most accurate I can think of. More logic could be added to the handler and the name would remain accurate.